### PR TITLE
util: Support getting MR ID from branch name

### PR DESCRIPTION
**Pull request by fmuellner**
If no MR ID is specified on the command line, the mr subcommands fall
back to looking for a MR associated with the current branch. Extend
that behavior to also try a branch name specified on the command line,
as branch names are usually easier to remember than MR numbers.

### CODE REVIEW

**Comment by prarit** [comment]
Ah this was on my list of TODOs :)

File: cmd/mr_edit.go [hunk 1]
```diff
@@ -22,36 +22,27 @@ var mrEditCmd = &cobra.Command{
 	Long:    ``,
 	Example: `lab MR edit <id>                                # update MR via $EDITOR
 lab MR update <id>                              # same as above
+lab MR update my-branch                         # same, but get MR ID from local branch
```
Comment (prarit) [inline cmd/mr_edit.go:25]: Nitpick -- maybe "my-branch-name" ?  Only change this if I find something else to complain about :)

**Review by prarit — APPROVED** [review]
LGTM

File: cmd/mr_edit.go (hunk 1 above)
Comment (bmeneguele) [reply cmd/mr_edit.go:25]: I would even suggest `<branch-name>`, but I know there are other help messages around that don't make use of `<>`, so... :))

**Review by bmeneguele — APPROVED** [review]

File: cmd/mr_edit.go (hunk 1 above)
Comment (fmuellner) [reply cmd/mr_edit.go:25]: Right, that would be more consistent with the `<id>` bit right above.

**Pull request merged**